changelog.json

```
@@ -1,5 +1,15 @@
 {
   "releases": {
+      "1.7.0-beta2": [
+        "[New] Rebase your current branch using a guided flow - #5953",
+        "[Fixed] Horizontal scroll bar visible in \"next steps\" section when switching branches - #7212",
+        "[Fixed] License templates do not end with newline character - #6999",
+        "[Fixed] Merge/Rebase conflicts banner do not clear when aborted outside Desktop - #7046",
+        "[Fixed] Missing tooltips for change indicators in the sidebar - #7174",
+        "[Fixed] Accessibility labels for icons would fail when multiple icons were visible at the same time - #7174",
```

Just changing this to match the tense of the other items.
```suggestion
        "[Fixed] Icon accessibility labels fail when multiple icons are visible at the same time - #7174",
```